### ISSUE TITLE
sdoc-merge: badges don't appear anymore

### ISSUE THREAD

**Issue by gamov** [voloko/sdoc#71]
I had to add:
```javascript
	result.badge = info[6]; 
```
After line: https://github.com/rdoc/rdoc/blob/master/lib/rdoc/generator/template/json_index/js/searcher.js#L174
for the badges to appear.
Since this file doesn't below to this gem BUT this modification is needed for this gem to work properly, I'm not sure how to proceed.

**Comment by zzak** [comment]
@gamov Sorry for the late response!

Yes, I have commit to RDoc, so this can be fixed upstream :)

Please re-open if you have any issues!

**Comment by zzak** [comment]
@gamov Can you try with rdoc master?

Just add `gem 'rdoc', github: 'rdoc/rdoc'` to your application's `Gemfile`, thanks!!

**Comment by gamov** [comment]
I don't use it within an app, but from the command line, this way:

     ./sdoc-merge --title "Ruby 1.9.3, Rails 3.0.20" --op merged  --names "rails, ruby" output/rails-3.0.20 output/ruby-1.9.3

Master triggered the following exception:
```
/Users/gamov/.rvm/rubies/ruby-1.9.3-p551/lib/ruby/1.9.1/rdoc.rb:101:in `const_missing': uninitialized constant RDoc::Generator (NameError)
	from /Users/gamov/Temp/@NoTM/gits/sdoc/lib/sdoc/generator.rb:94:in `<top (required)>'
	from /Users/gamov/.rvm/rubies/ruby-1.9.3-p551/lib/ruby/site_ruby/1.9.1/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/gamov/.rvm/rubies/ruby-1.9.3-p551/lib/ruby/site_ruby/1.9.1/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/gamov/Temp/@NoTM/gits/sdoc/lib/sdoc.rb:7:in `<top (required)>'
	from /Users/gamov/.rvm/rubies/ruby-1.9.3-p551/lib/ruby/site_ruby/1.9.1/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/gamov/.rvm/rubies/ruby-1.9.3-p551/lib/ruby/site_ruby/1.9.1/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/gamov/Temp/@NoTM/gits/sdoc/bin/sdoc-merge:2:in `<main>'
```
I have actually switched to Dash (https://kapeli.com/dash) for my documentation needs, it's great and merges in realtime.
